src/WSRequestHandler.cpp

```diff
@@ -87,7 +87,11 @@ QHash<QString, HandlerResponse(*)(WSRequestHandler*)> WSRequestHandler::messageM
 	{ "GetSourceSettings", WSRequestHandler::HandleGetSourceSettings },
 	{ "SetSourceSettings", WSRequestHandler::HandleSetSourceSettings },
 	{ "TakeSourceScreenshot", WSRequestHandler::HandleTakeSourceScreenshot },
-
+	{ "AddNewSource", WSRequestHandler::HandleCreateNewSource },
```

Comment:
Inconsistent naming: `AddNewSource` vs. `CreateNewSource`. This got me. I sent a web request "CreateNewSource", only to be greeted by "invalid request type", since the request type is `AddNewSource`.